bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/FormContainer.java

```diff
@@ -373,4 +373,26 @@ default void visit(Consumer<ComponentExporter> callback) throws Exception {}
     default String getParentPagePath() {
         return null;
     }
+
+    /**
+     * Returns the url from where the custom functions should be registered
+     * 
+     * @return custom function registration url
+     * @since com.adobe.cq.forms.core.components.models.form 5.7.6
+     */
+    @JsonIgnore
+    default String getCustomFunctionAction() {
```

Comment:
Please refer to the implementations of getAction and getDataUrl; there should be no property for this in JCR. For getPrefillAction, let's reuse the existing data URL implementation and rename it to getCustomFunctionURL.